docker-compose.acra-censor-demo.yml

```diff
@@ -1,138 +1,77 @@
-            --keys_dir=/keys
-            --auth_keys=/keys/httpauth.accounts
-            --http_api_enable
-            --incoming_connection_api_string=tcp://0.0.0.0:9090
+            --keystore_cache_on_start_enable=false
             --acracensor_config_file=/config/acra-censor.yaml
-            -v
+            --v
+            --d
 
+    #===== OWASP Mutillidae II =================================================
 
-    acra-connector:
-        image: "cossacklabs/acra-connector:${ACRA_DOCKER_IMAGE_TAG:-latest}"
-        restart: always
+    mutillidae:
+        container_name: mutillidae
         depends_on:
-            - acra-keymaker_server
-            - acra-keymaker_connector
+            - database
             - acra-server
-        # Open the port outside for client application
+        image: mutillidae
+        build:
+            context: https://github.com/webpwnized/mutillidae-docker.git#1.0.39:www
+            args:
+                DATABASE_HOST: acra-server
+                DATABASE_PORT: 9393
         ports:
-            - "3306:3306"
+            - 127.0.0.1:8080:80
         networks:
-            - acraconnector-acraserver
-            - mutillidae-acraconnector
-        environment:
-            ACRA_MASTER_KEY: ${ACRA_MASTER_KEY:-UHZ3VUNNeTJ0SEFhbWVjNkt4eDdVYkc2WnNpUTlYa0E=}
-        volumes:
-            # Mount the directory with only the keys for this service
-            - ./.acrakeys/acra-connector:/keys:ro
-        command: >-
-            --acraserver_connection_host=acra-server
-            --keys_dir=/keys
-            --client_id=${ACRA_CLIENT_ID:-testclientid}
-            --incoming_connection_string=tcp://0.0.0.0:3306
-            --http_api_enable
-            --incoming_connection_api_string=tcp://0.0.0.0:9191
-            -v
-
-
-    #===== OWASP Mutillidae II =================================================
-
-    edoz90_mutillidae:
-        # Build base image
-        build:
-            context: https://github.com/storojs72/docker-mutillidae.git
-        image: storojs72/edoz90_mutillidae:latest
-        # We don't need to run the container based on the original image
-        entrypoint: /bin/true
-        restart: 'no'
+            - world
+            - mutillidae-acraserver
 
+    #===== OWASP Mutillidae DB =================================================
 
-    mutillidae:
-        depends_on:
-            - acra-connector
-            - edoz90_mutillidae
+    database:
+        container_name: database
+        image: webpwnized/mutillidae:database
+        healthcheck:
+            test: "/usr/bin/mysql --user=root --password=mutillidae --execute \"SHOW DATABASES;\""
+            interval: 5s
+            timeout: 30s
+            retries: 10
         build:
-            context: ./mutillidae
-        image: mutillidae:latest
-        ports:
-            - "8080:80"
-            - "3306:3306"
+            context: https://github.com/webpwnized/mutillidae-docker.git#1.0.39:database
```

Comment:
do we really need this context? looks like it's empty folder with only dockerfile